src/Project.js

```diff
@@ -473,14 +475,53 @@ class Project extends DirItem {
      * @returns {Array.<Result>} a list of results
      */
     findIssues(locales) {
+        this.fileStats = new FileStats();
         return this.files.map(file => {
             logger.trace(`Examining ${file.filePath}`);
 
-            file.parse();
+            const irArray = file.parse();
+            if (irArray) {
+                irArray.forEach(ir => {
+                    if (ir.stats) {
+                        this.fileStats.addStats(ir.stats);
+                    } else {
+                        // no stats? At least we know there was a file, so count that
+                        this.fileStats.addFiles(1);
+                    }
+                });
+            }
             return file.findIssues(locales);
         }).flat();
     }
 
+    /**
+     * Return the I18N Score of this project. The score is a number from
+     * zero to 100 which gives the approximate localization readiness of
+     * the whole project. The absolute number of the score is not as
+     * important as the relative movement of the score, as the increase
+     * in score shows an improvement in localizability.
+     *
+     * In this particular score, errors are weighted most heavily,
+     * followed by warnings at a medium level, and suggestions at a
+     * very light level.
+     *
+     * @returns {Number} the score (0-100) for this project.
+     */
+    getScore() {
+        if (!this.fileStats) {
+            throw "Attempt to calculate the I18N score without having retrieved the issues first.";
+        }
+
+        const base = (this.fileStats.modules || this.fileStats.lines || this.fileStats.files);
+        const demeritPoints = this.resultStats.errors * 5 + this.resultStats.warnings * 3 + this.resultStats.suggestions;
+
+        // divide demerit points by the base so that larger projects are not penalized for
+        // having more issues just because they have more files, lines, or modules
+        // y intercept = 100
+        // lim(x->infinity) of f(x) = 0
+        return 100 / (1.0 + demeritPoints/base);
```

Comment:
probably will need rounding to int

also I think prior to calculations we should add a check `if (!(base > 0)) { throw new Error("cannot calculate error because base file stat is zero"); }`, so that it would be more meaningful than just _error due to division by zero_

Reply:
rounding -> There is a formatter on line 546 that limits it to 2 decimal places

Added a default value of 1 for base